db/seeds/languages

```diff
@@ -0,0 +1,336 @@
+INSERT INTO languages VALUES (166, 'Mbudum', '1', 'xmd', 8, '', 546, 'Francophone', 6000, '6,000 (2001 SIL).', 'Afro-Asiatic', 4, '2017-05-04 10:15:19.389891', '2020-03-31 23:16:03.080765', 'Boudoum, Ma Mbudum, Mbedam, Mboudoum', NULL, 6, NULL, 170);
+INSERT INTO languages VALUES (367, 'Gemzek', NULL, NULL, NULL, NULL, 546, NULL, NULL, NULL, NULL, 4, '2017-07-28 13:03:01.529701', '2020-03-25 13:39:28.15763', NULL, 278, 5, NULL, 370);
+INSERT INTO languages VALUES (61, 'Buwal', '1', 'bhs', 6, 'Buwa/Gavar RTT done Feb2000.  Buwal def. Need with possibly inclusion of Gavar.Eth14: May be intelligible with Gavar.  Speakers closer to Mofu or Gavar regions claim to understand those languages. Fulfulde and French bilingualism is limited.  Buwal is', 546, 'Francophone', 10000, '10,000 (2004 SIL), increasing.', 'Afro-Asiatic', 4, '2017-05-04 10:15:17.411788', '2020-03-31 23:06:30.471728', 'Bual, Gadala, Ma Buwal', NULL, 6, NULL, 68);
+INSERT INTO languages VALUES (263, 'Vame', '1', 'mlr', 6, 'Potential future secondary project.   Mo Perrin may help if Andre Doucha is free to translate in his own language.     A standard reference dialect needs to be identified, but Mindreme seems probable.      3 Word lists available (CREA).     Willie K', 546, 'Francophone', 8500, '8,500 (1992 SIL). Few monolinguals.', 'Afro-Asiatic', 4, '2017-05-04 10:15:21.18463', '2020-04-03 08:10:02.397347', 'Maslava, Ndreme, Pelasla', NULL, 5, NULL, 267);
+INSERT INTO languages VALUES (12, 'Bafanji', '1', 'bfj', 6, 'RTT should wait until one of the Ndop Plain languages is being standardized.Eth14: Speakers refer to their language as ''Chuufi''.  Closely related to Bamali, Bamenyam, and Bambalang', 546, 'Anglophone', 17000, '17,000 (2008 SIL).', 'Niger-Congo', 7, '2017-05-04 10:15:16.509365', '2020-04-03 21:51:47.558629', 'Bafangi, Bafanyi, Chuufi, Nchufie', NULL, 3, NULL, 19);
+INSERT INTO languages VALUES (29, 'Bamukumbit', '1', 'bqt', 6, 'RTT should wait until one of the Ndop Plain languages is being standardized.Bamenkubit est uotidiennement utilisee et ne connait pas l''influence d''une autre langue. Son appartenance aux parlers de ses voisins (soit Bafanji, Bambalang, Bamali et Bameny', 546, 'Anglophone', 12000, '12,000 (2008 SIL).', 'Niger-Congo', 7, '2017-05-04 10:15:16.812527', '2020-04-03 22:01:34.206947', 'Bamenkombit, Bamenkoumbit, Bamoukoumbit, Bamunkum, Bamunkumbit, Mangkong, Mankong', NULL, 3, NULL, 36);
+INSERT INTO languages VALUES (123, 'Kemedzung', '1', 'dmo', 6, 'est. pop: 4,400RTT to measure comprehension of Nsari.', 546, 'Anglophone', 3540, '3,540 (2008 Misaje council).', 'Niger-Congo', 7, '2017-05-04 10:15:18.575675', '2020-04-14 14:50:23.265329', 'Dumbo, Dzumbo, Kemezung, Kumaju', NULL, 1, NULL, 127);
```

Comment:
A lot of random notes in here that maybe don't need to be public information. Perhaps a private repo is what we really need...

Reply:
I'll simply the seeded data. On a quick look through I thought that was just Ethnologue data.